src/test/java/net/sf/javaanpr/configurator/ConfiguratorTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2013 JavaANPR contributors
+ * Copyright 2006 Ondrej Martinsky
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.osedu.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package net.sf.javaanpr.configurator;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.awt.*;
+import java.io.File;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConfiguratorTest {
+    private Configurator configurator;
+    private final String propertyName = "new property";
+
+    @Before
+    public void setup() {
+        configurator = Configurator.getConfigurator();
```

Comment:
The `Configurator`, as it is written now, is pretty bad. The fact that it's a singleton could potentially cause quite a few issues, for example with testing. Another person could come in, create a test that assumes that the "new property" property is empty, because we get a "new" configurator before every test in the setup method, which is not really the case.

I'll merge this, as the test is good, and attempt a fix later this week. If you want, you can create an issue for this.